findoutlie/metrics.py

```diff
@@ -54,7 +54,42 @@ def dvars(img):
     # You may be be able to solve this in four lines, without a loop.
     # But solve it any way you can.
     data = img.get_fdata()
-    voxel_per_time = np.reshape(data, (-1, data.shape[-1]))
+    voxel_per_time = np.reshape(data, (-1, data.shape[-1])) #np.reshape(data,new_shape)
     diff = np.diff(voxel_per_time)
     dvals = np.sqrt(np.mean(diff ** 2, axis=0))
     return dvals
+
+
+#def standardized_dvar(img):
+    #https: // warwick.ac.uk / fac / sci / statistics / staff / academic - research / nichols / scripts / fsl / standardizeddvars.pdf
+
+
+def coefficient_of_variation(img):
+    #looking at the output nii I am not quite sure this is right
+    """ Calculate the coefficient of variation (CV)
+    also known as relative standard deviation (RSD),
+    is a standardized measure of dispersion of a probability distribution
+    or frequency distribution. It is often expressed as a percentage,
+    and is defined as the ratio of the standard deviation to the mean
+
+    NB : Issue if  mean value is close to zero, should not be the case though
+    TO DO ? : would be better to do it for each tissue class
+    or at least on the brain vs background, not sure it makes a lot of sense here
+
+    Parameters
+    ----------
+    img : nibabel image
+
+    Returns
+    -------
+    CV : 1D array
+        One-dimensional array with n elements, where n is the number of
+        volumes in `img`. This array contains the coefficient of variation for each volume.
+
+    """
+    data = img.get_fdata()
+    CV=np.std(data, axis=(0,1,2))/np.mean(data, axis=(0,1,2))
+
+    return CV
```

Comment:
```suggestion
    cv=np.std(data, axis=(0,1,2))/np.mean(data, axis=(0,1,2))

    return cv
```

I would suggest not to name variables with capital letters